test/e2e/accounts/snap-account-contract-interaction.spec.ts

```diff
@@ -1,4 +1,4 @@
-import GanacheContractAddressRegistry from '../seeder/ganache-contract-address-registry';
+import ContractAddressRegistry from '../seeder/contract-address-registry';
```

Comment:
just a rename, as not it's used with ganache and anvil - no functionality change